### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by zmLGBBM** [AtomLinter/linter-clang#65]
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (C:\Users\Ming\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:790:9)
Package.hasActivationCommands (C:\Users\Ming\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:715:20)
<unknown> (C:\Users\Ming\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:169:24)
Package.measure (C:\Users\Ming\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:147:15)
```

**Comment by k2b6s9j** [comment]
I fixed this as well. Still waiting on @Snake231088 to push the patches to APM.

**Comment by kevinsawicki** [comment]
Fixed by https://github.com/AtomLinter/linter-clang/pull/61 and published in version `2.25.2`

**Comment by zmLGBBM** [comment]
@kevinsawicki Great, Thanks!
